draft-irtf-cfrg-vdaf.md

```diff
@@ -6153,11 +6131,12 @@ class QueryGadget(Gadget[F]):
 
 # Test Vectors {#test-vectors}
 
-The test vectors are available at {{TestVectors}}. The test vector directory,
-`test_vec/vdaf`, contains a set of JSON files. Each file contains a test vector
-for an instance of `Vdaf` ({{vdaf}}). A test vector covers sharding,
-preparation, aggregation, and unsharding of each of several measurements. The
-test vector schema is defined below.
+Test vectors for Prio3 ({{prio3}}) and Poplar1 ({{poplar1}}) are available at
+{{TestVectors}}. The test vector directory, `test_vec/vdaf`, contains a set of
+JSON files. Each file contains a test vector for an instance of class `Vdaf` as
+defined in {{vdaf}}. A test vector covers sharding, preparation, aggregation,
+and unsharding of each of several measurements. The test vector schema is
```

Comment:
nit: This may be a bit clearer, since the aggregation and unsharding operate on the batch as a whole.
```suggestion
and unsharding of a batch of several measurements. The test vector schema is
```